valenstical/ireporter
# #162196969 Fixes dialog scrolling issue on admin page

**What does this PR do?**
* This PR fixes the dialog scrolling issue for the admin page on mobile devices.

**Description of tasks**

* Add extra right margin to the dialog overlay to make room for the vertical scrollbar. 

**How can this PR be tested**

* Visit https://valenstical.github.io/ireporter/admin-reports.html on a mobile device
* Click on any of the reports listed on the table
* The dialog appears with an overlay that covers the screen with space for the vertical scrollbar   

**What are the relevant pivotal tracker stories?**
[#162196969 ](https://www.pivotaltracker.com/story/show/162196969)

**Screenshot**
N/A
